scripts/test-react.js

```diff
@@ -158,7 +165,7 @@ function runTestSuite(outputJsx) {
   // Jest tests
   let originalConsoleError = console.error;
 
-  describe(`Test React (${outputJsx ? "JSX" : "create-element"})`, () => {
+  describe(`Test ${shouldTranspileSource ? "transpiled" : ""} React (${outputJsx ? "JSX" : "create-element"})`, () => {
```

Comment:
What is the difference between "transpiled JSX" and "create-element"?

Reply:
Transpiled source is when the code is run through Babel before being passed to Prepack to be evaluated and compiled.

Reply:
Ah I get it now. Maybe it would be clearer if it said something like "JSX input, JSX output" or "non-JSX input, non-JSX output" etc. Right now it's not clear what "JSX" refers to in the test name.